server/src/db/seeders/TripSeeder.js

```diff
@@ -27,12 +29,14 @@ class TripSeeder {
         numberOfDays: "10",
         description:
           "I love Cape Town! It has everything you want: great hikes with beautiful views, amazing people and some of the best food I've ever had",
+        userId: 3,
       },
       {
         continent: "Asia",
         country: "Israel",
         title: "Fun place",
         description: "Great food, nice people, awesome night life",
+        userId: 2,
```

Comment:
Because I've seen this cause issues for other groups in the past: I'd recommend querying users at the top of this file, then calling `userId: user1.id` instead of hardcoding `userId: 1` so that you can trust the seeder will work regardless of deleted users.